config/settings.py

```diff
@@ -194,6 +194,58 @@
     "DEFAULT_PERMISSION_CLASSES": [],
 }
 
+
+# Build server URLs dynamically for OpenAPI documentation
+def get_api_servers():
+    """
+    Generate server configuration for OpenAPI schema based on environment.
+    This ensures that Fumadocs and other API documentation tools have proper
+    server endpoints for testing the API.
+    """
+    servers = []
+
+    # Get the API port configuration
+    api_port = os.getenv("API_PORT", "127.0.0.1:8000")
+
+    if DEBUG:
+        # Development servers
+        if ":" in api_port:
+            host, port = api_port.split(":")
+            dev_url = f"http://{host}:{port}/api"
+        else:
+            dev_url = f"http://localhost:{api_port}/api"
+
+        servers.append({"url": dev_url, "description": "Development server"})
+
+        # Also add localhost variant for Docker development
+        servers.append(
+            {
+                "url": "http://localhost:8000/api",
+                "description": "Local Docker development server",
+            }
+        )
+    else:
+        # Production server
+        production_url = os.getenv("PRODUCTION_API_URL")
```

Comment:
The URL of the production server is not going to change, there is no need to use an envvar here.
You can your code by directly adding `https://prices.openfoodfacts.org/api`.